README.rst

```diff
@@ -1,7 +1,26 @@
+flit
```

Comment:
The docs page which includes this already has the logo in: http://flit.readthedocs.io/en/latest/

If the README includes the logo as well, the docs would have it twice.

But since we can't right-align it in the README, I'd rather drop the logo from the README so that the useful text starts sooner, and leave it just in the docs, where we can right-align it.